src/EnergyPlus/Data/EnergyPlusData.hh

```diff
@@ -109,16 +132,38 @@ struct ExteriorEnergyUseData : BaseGlobalStruct
         }
     };
 
+    struct ExteriorEquipmentUsage

```

Comment:
So this is the big question, I think.  What do we do with structs?   Ahead of this refactor, it is pretty simple, structs and function declarations go in the matching header with the code.  But with this refactor, we have a choice.  We can either move the structs out to the data folder or we can leave it in the original header and forward declare the EnergyPlus data struct instance.  The main pro/con I see is that if we start pulling headers into this file, the file is going to quickly grow to a very large size.  Like very.  I don't really see any problems with forward declaring the data struct so that we can leave the headers in the original location.  Another perk about this approach is that the files changed on Github will go down too, so it will be easier to review.  I'm certainly open for discussion, but that's the way I'm leaning right now.  Let me know if you want me to take a quick pass here and apply that.